unit-tests/live/safety_preset/test-read-write-hwm-cmd.py

```diff
@@ -0,0 +1,76 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+# test:device S585
+
+import pyrealsense2 as rs
+from rspy import devices, log, test, file, repo
+
+#############################################################################################
+# Tests
+#############################################################################################
+
+test.start("Init")
```

Comment:
remove try

Reply:
done

Reply:
If you decided to rename this on the other py align here, if not resolve :)